src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -63,6 +64,10 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
 }) => {
   const { theme } = useTheme();
   const { i18nt } = useTranslate();
+  const recordedDate = {
+    dateTime:
+      lastTransactionTime !== null ? format(lastTransactionTime, "h:mma") : "-",
+  };
```

Comment:
I noticed you've extracted out `recordedDate` to an object. However, the purpose of the final object param of `i18nt` is to provide values for the string substitutions. Perhaps we can extract out the dateTime instead?
e.g. (ignore the formatting.. typing this on github)
```
const formattedLastTransactionTime = lastTransactionTime !== null ? format(lastTransactionTime, "h:mma") : "-";
...
{i18nt(
              "statisticsScreen",
              theme.name === GOVWALLET_THEME_NAME ? "lastRecordedTiming" : "lastDistributedTiming",
              undefined,
             {
               dateTime: formattedLastTransactionTime
             }
            )
          }

```